options/locale/locale_en-US.ini

```
@@ -733,6 +733,8 @@ issues.action_milestone_no_select = No milestone
 issues.action_assignee = Assignee
 issues.action_assignee_no_select = No assignee
 issues.opened_by = opened %[1]s by <a href="%[2]s">%[3]s</a>
+issues.merged_by = merged %[1]s by <a href="%[2]s">%[3]s</a>
```

Maybe move that to pulls ?
```suggestion
pulls.merged_by = merged %[1]s by <a href="%[2]s">%[3]s</a>
```

---

We could also add a pulls.closed_by or pulls.cancelled_by as a closed PR not merged is kind of cancelled and the if/else checks can provide this distinction allready.

---

I think it should be left as `issues.closed_by` since it is used by both issues and PRs. I don't think it is ideal we have both `issues.closed_by` and `pulls.closed_by`. It will also mean more ___if/elses____ to check which translation string to use 

I committed your suggestion for `pulls.merged_by`

---

@adelowo It's just a suggestion for the cancel one. 
Don't forget to rename in the template also and re-order to the pulls part.